package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "fastify-ws",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

I'll bump the package version in a separate commit
```suggestion
  "version": "1.0.0",
```